index.html

```diff
@@ -422,10 +423,10 @@ <h2>Selection interface</h2>
 
         <dt><dfn>deleteFromDocument</dfn></dt>
         <dd>
-          <p>The method must invoke <dfn><a href="https://dom.spec.whatwg.org/#dom-range-deletecontents"><code>deleteContents()</code></a></dfn> ([[!DOM]])
+          <p>The method must invoke <dfn><a href="!DOM#dom-range-deletecontents"><code>deleteContents()</code></a></dfn> ([[!DOM]])
```

Comment:
Should be data-cite :) ... don't need ! anymore... will update those in a different PR. 
```suggestion
          <p>The method must invoke <dfn><a data-cite="DOM#dom-range-deletecontents"><code>deleteContents()</code></a></dfn> ([[DOM]])
```